### DISCUSSION TITLE
Using this project's bug I think they have my bot's token

### DISCUSSION THREAD

**Discussion by ynsemrekaya** [Sayrix/Ticket-Bot#132] [General]

**Comment by Sayrix** [comment]
Hi, it's impossible that they had access to the bot's token with the Ticket-Bot because the token is not sent to any service

**Comment by thomasbnt** [comment]
Hello, if you want to reset your token of your bot, 
- https://discordapp.com/developers/applications/me
- Select the bot concerned
- Go to _Bot_ section
- And click on the **Reset Token** button

![How to reset a Discord bot token](https://user-images.githubusercontent.com/14293805/229366766-951c3c50-02dc-41a8-ad5a-baee8a7423bc.png)

**Comment by BlueSky-fur** [comment]
> Due to a security vulnerability, a few people maliciously trying to delete all channels on my server, where I have 10,000 members, and remove members from the server. I didn't share my bot's token anywhere, I was keeping it a secret.
> 
> ![image](https://user-images.githubusercontent.com/104264467/229331587-c0ac3876-924c-469a-a49f-928a9c2f9df4.png)

The Sourcecode is completely open, as long as you download it from **here** you are always free to view the code inside of it.
**Nothing** regarding any Tokens gets sent out.
